.changeset/chatty-ears-hammer.md

```diff
@@ -0,0 +1,5 @@
+---
+'@guardian/source-development-kitchen': major
+---
+
+New ticker component
```

Comment:
As this is new functionality and won't break any existing uses, i think you can use a `minor` here.

```suggestion
---
'@guardian/source-development-kitchen': minor
---

Add a new `ticker` component
```